CONTRIBUTING.md

```diff
@@ -0,0 +1,14 @@
+## Publishing
+
+Make sure your repo is up to date with the latest master from upstream.
+To publish a new minor release to NPM:
+
+```sh
+npm version minor
```

Comment:
You can reference this article

https://tstringer.github.io/npm/npmjs/2015/10/29/npm-incrementing-version.html

To explain npm version patch, minor, major.